backend/app/routers/admin.py

```
@@ -1,12 +1,26 @@
-from utils import get_current_admin
-from schemas import UserUpdate, ProductUpdate, OrderUpdate
-from schemas import PromoteUser
+from app.models import User, Product, Order
+from app.database import get_db
+from app.utils import get_current_admin
+from app.schemas import UserUpdate, ProductUpdate, OrderUpdate, PromoteUser
+
+router = APIRouter()
+
+
+def get_object_or_404(model, object_id: int, db: Session, field="id", name="Item"):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_object_or_404`

(ANN201)